tests/test_interface.py

```diff
@@ -27,27 +27,6 @@ def test_outputs():
         assert otype == expected_types[i]
 
 def test_family_closure():
-    #family_closure needs single argument
-    with pytest.raises(SyntaxError):
```

Comment:
If you want to keep these tests you can use `pytest.warns` 
https://docs.pytest.org/en/latest/warnings.html#warns